gotestcover_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func TestParseFlags(t *testing.T) {
+	os.Args = []string{"gotestcover",
```

Comment:
Good point, I will check that. I assume as `go test` is already running, it shouldn't but I will make some checks with existing parameters.